lib/ansible/modules/cloud/azure/azure_rm_storageaccount.py

```diff
@@ -420,8 +444,8 @@ def create_account(self):
 
     def delete_account(self):
         if self.account_dict['provisioning_state'] == self.storage_models.ProvisioningState.succeeded.value and \
-           self.account_has_blob_containers() and self.force:
-            self.fail("Account contains blob containers. Is it in use? Use the force option to attempt deletion.")
+           self.account_has_blob_containers() and self.force_delete_nonempty:
```

Comment:
if force_delete_nonempty true, why still fail when blob containers? naming of argument means it will be force deleted if not empty, right?